extensions/amp-minute-media-player/0.1/test/test-amp-minute-media-player.js

```diff
@@ -42,65 +43,139 @@ describes.realWin(
+      player.setAttribute('width', WIDTH);
+      player.setAttribute('height', HEIGHT);
+      player.setAttribute('layout', RESPONSIVE);
 
-      doc.body.appendChild(minuteMediaPlayerElement);
-      return minuteMediaPlayerElement
+      doc.body.appendChild(player);
+      return player
         .buildInternal()
         .then(() => {
-          minuteMediaPlayerElement.layoutCallback();
+          player.layoutCallback();
         })
-        .then(() => minuteMediaPlayerElement);
+        .then(() => player);
     }
 
-    it('renders with curated content', async () => {
-      const minuteMediaPlayerElement = await getMinuteMediaPlayer({
-        'data-content-type': CURATED,
-        'data-content-id': DATA_CONTENT_ID,
+    describe('rendering', async () => {
+      it('renders with curated content', async () => {
+        const player = await getMinuteMediaPlayer({
+          'data-content-type': CURATED,
+          'data-content-id': DATA_CONTENT_ID,
+        });
+        const iframe = player.querySelector('iframe');
+        expect(iframe).to.not.be.null;
+        expect(iframe.src).to.equal(
+          `https://www.oo-syringe.com/prod/AMP/minute-media-player.html?content_type=${CURATED}&content_id=${DATA_CONTENT_ID}`
+        );
+        expect(iframe.className).to.match(/i-amphtml-fill-content/);
       });
-      const iframe = minuteMediaPlayerElement.querySelector('iframe');
-      expect(iframe).to.not.be.null;
-      expect(iframe.src).to.equal(
-        `https://www.oo-syringe.com/prod/AMP/minute-media-player.html?content_type=${CURATED}&content_id=${DATA_CONTENT_ID}`
-      );
-      expect(iframe.className).to.match(/i-amphtml-fill-content/);
-    });
 
-    it('renders with semantic (empty params)', async () => {
-      const minuteMediaPlayerElement = await getMinuteMediaPlayer({
-        'data-content-type': SEMANTIC,
-        /* no params to semantic */
+      it('renders with semantic (empty params)', async () => {
+        const player = await getMinuteMediaPlayer({
+          'data-content-type': SEMANTIC,
+          /* no params to semantic */
+        });
+        const iframe = player.querySelector('iframe');
+        expect(iframe).to.not.be.null;
+        expect(iframe.src).to.equal(
+          `https://www.oo-syringe.com/prod/AMP/minute-media-player.html?content_type=${SEMANTIC}`
+        );
+        expect(iframe.className).to.match(/i-amphtml-fill-content/);
+      });
+
+      it('renders with semantic (with params)', async () => {
+        const player = await getMinuteMediaPlayer({
+          'data-content-type': SEMANTIC,
+          'data-minimum-date-factor': DATA_MINIMUM_DATE_FACTOR,
+          'data-scanned-element-type': DATA_SCANNED_ELEMENT_TYPE,
+          'data-scoped-keywords': DATA_SCOPED_KEYWORDS,
+        });
+        const iframe = player.querySelector('iframe');
+        expect(iframe).to.not.be.null;
+        expect(iframe.src).to.equal(
```

Comment:
nit: verify that `iframe.src` contains SEMANTIC, DATA_SCANNED_ELEMENT_TYPE, etc.

Reply:
Sorry, my earlier comment wasn't clear. I should have mentioned "verify that `iframe.src`... instead of verifying the entire URL". You basically care about the iframe URL containing SEMANTIC, DATA_MINIMUM_DATE_FACTOR, etc. The rest of the URL is constant.

Reply:
Ah! Thanks for the clarification I'll make that change